offchain-modules/packages/scripts/package.json

```diff
@@ -10,6 +11,9 @@
   },
   "dependencies": {
     "@force-bridge/x": "*",
-    "@force-bridge/xchain-eth": "*"
-  }
+    "@force-bridge/xchain-eth": "*",
```

Comment:
Some of the dependencies is `^0.0.0`, some are `*`, shall we use the same one?

Reply:
> Some of the dependencies is `^0.0.0`, some are `*`, shall we use the same one?

Before publish, `*` and `0.0.0` are both local versions, there is not much difference. It will changed to `0.0.1` after we first publish it